File: code_samples/php/v1.0.0/cancel_payment.php
<?php
use Tatrapayplus\TatrapayplusApiClient\Api\TatraPayPlusAPIApi;

$tatrapayplus_api = new TatraPayPlusAPIApi(
    "your-client-id",
    "your-client-secret",
);
$payment_id = 'b54afd37-5bb9-4080-9416-5ec450779087'; // Retrieved from initiatePayment

$response = $tatrapayplus_api->cancelPaymentIntent($payment_id);

File: code_samples/python/v1.0.0/cancel_payment.py
from tatrapayplus import TBPlusSDK

client = TBPlusSDK(
    "your-client-id",
    "your-client-secret"
)
payment_id = "b54afd37-5bb9-4080-9416-5ec450779087"  # Retrieved from create payment intent
cancel_payment_response = client.cancel_payment(payment_id)


